temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
+    with open(filepath, "w") as output_file:
+        if file_type == "txt":
+            output_file.write("")
+        elif file_type == "json":
+            output_file.write(json.dumps({}))
+        else:
+            log.error("ERROR: Invalid file type")
```

Comment:
**Suggestion:** Import the 'json' module to ensure that 'json.dumps({})' works. Without this import, the code will raise a 'NameError' when 'file_type' is 'json'. [bug]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
import json

def reset_files(filepath, file_type="txt"):
    with open(filepath, "w") as output_file:
        if file_type == "txt":
            output_file.write("")
        elif file_type == "json":
            output_file.write(json.dumps({}))
        else:
            log.error("ERROR: Invalid file type")
```